### DISCUSSION TITLE
DNS disappears after update

### DISCUSSION THREAD

**Discussion by testQaBox** [elementary/triage#290] [Q&A]
Could you help me figure out what is the reason or step by step how to determine the cause of the loss of DNS
After installing the operating system, everything works, but after updates and reboots DNS stops working
This applies to both wired and non-wired networks.

--> resolvectl status
Global
       Protocols: -LLMNR -mDNS -DNSOverTLS DNSSEC=no/unsupported
resolv.conf mode: stub

Link 2 (wlo1)
Current Scopes: none
     Protocols: -DefaultRoute +LLMNR -mDNS -DNSOverTLS DNSSEC=no/unsupported

Link 3 (ipv6leakintrf0)
    Current Scopes: DNS
         Protocols: +DefaultRoute +LLMNR -mDNS -DNSOverTLS DNSSEC=no/unsupported
Current DNS Server: ::1
       DNS Servers: ::1
        DNS Domain: ~.

When trying to set the settings in the user interface, they also give no results.

Trying to set preferences in the UI (Set DNS (f.e. 8.8.8.8, 8.8.4.4) also fails.
If I'm not mistaken, then the installed DNS settings should also be displayed in the resolv.conf files?

Could you please provide a debugging map of this problem and a possible solution to this problem?

**Comment by RogerWeihrauch** [comment]
hi
hm, on the first look, with regards to your given informations, it seems to me that:
1) actually,after update: you only have IPv6 at hand
2) maybe you should try to (re-) activate IPV4 first, get it up and running and than try to switch completely to IPV6
3) maybe your DHCP Server (Router) only distributes IPv6 to its clients? Try to check
4) also try to check the settings on your Elementary install for its network cards: are they restricted to IPv6?
Regards,
Roger
